### DISCUSSION TITLE
Welcome to Obsidian Discussions!

### DISCUSSION THREAD

**Discussion by Naamloos** [Naamloos/Obsidian#70] [General]
## 👋 Welcome to Obsidian's GitHub Discussions
In this forum we will allow sharing content about Obsidian, Discussion about the server, and Plugin Releases.

Feel free to add some requests for the forum below!

**Comment by roxxel** [comment]
Ok. 🙃

**Comment by developer-ken** [comment]
Wow nothing to discuss but what a cool ideal to impliment the whole thing in C#!
love the project
